pages/home.page.tsx

```diff
@@ -78,33 +82,78 @@ export default function HomePage({ navigation }: IReactPageServices) {
     );
   };
 
+
+  
+
   React.useLayoutEffect(() => {
     navigation.setOptions({
       headerRight: () => (
         <View style={{ flexDirection: 'row' }} onTouchStart={() => showPage('profilePage')}>
-          <Icon.Button backgroundColor="transparent" underlayColor="transparent" color={themePalette.shellNavColor} name='cog-outline' />
-        </View>
+        {/* <Image source={require('../assets/settings.png')} style={{ width: 24, height: 24 }} /> */}
+        <IconIonicons name="settings-outline" color={colors.white} size={24} />
+    
+      </View>
       )
     });
   });
 
   return (
     <Page >
-      <StatusBar style="auto" />
-      <View style={{ width: "100%" }} >
-        <Image style={[{ marginTop: 30, marginBottom: 30, alignSelf: "center" }]} source={require('../assets/app-icon.png')} />
-        <Text style={[{ textAlign: 'center', marginBottom: 5, color: themePalette.shellTextColor, fontSize: 24 }]}>{user?.currentOrganization.text} Instances</Text>
+      {/* <StatusBar style="auto" /> */}
+      <View style={{padding: 16, width: "100%", height: "100%", backgroundColor: themePalette.background }} >
+        {/* <Image style={[{ marginTop: 30, marginBottom: 30, alignSelf: "center" }]} source={require('../assets/logo.png')} /> */}
+        {themePalette.name == 'light' ? (
+          <Image style={[{ marginTop: 30, marginBottom: 30, alignSelf: "center" }]} source={require('../assets/logo.png')} />
+        ) : (
+          <Image style={[{ marginTop: 30, marginBottom: 30, alignSelf: "center"}]} source={require('../assets/logo-dark-theme.png')} />
```

Comment:
These images appear to have the words "Software Logistics Instance" on them.  That text should be populated from text as it was before